src/main/java/apaintus/models/ListViewCell.java

```diff
@@ -0,0 +1,48 @@
+package apaintus.models;
```

Comment:
Je ne pense pas que cette classe devrait être là. @pgarneau javafx ne devrait pas etre dans models, right?

Reply:
Oops, forgot about that

Reply:
@boscoa J'ai fait le meme comment :P